app/admin/realtime_data/sip_options_probers.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+ActiveAdmin.register RealtimeData::SipOptionsProber, as: 'Sip Options Probers' do
+  menu parent: 'Realtime Data', label: 'Sip Options Probers', priority: 10
+  config.batch_actions = false
+
+  actions :index
+
+  controller do
+    def scoped_collection
+      RealtimeData::SipOptionsProber.all
+    end
+
+    def apply_sorting(chain)
+      chain
+    end
+
+    def apply_filtering(chain)
+      query_params = (params.to_unsafe_h[:q] || {}).delete_if { |_, v| v.blank? }
+      @search = OpenStruct.new(query_params)
+      chain = chain.none if query_params.blank?
```

Comment:
we should show all probers when no filters assigned

currently empty table will be shown